lib/ui/painting.dart

```diff
@@ -399,6 +423,7 @@ class Paint {
     _data.setInt32(_kColorOffset, encoded, _kFakeHostEndian);
   }
 
+  // Must be kept in sync with the default in paint.cc
```

Comment:
ok i feel like i'm just yanking your chain now but you put the period on the comment above and didn't here...